main.tf

```
@@ -153,65 +63,37 @@ module "log_storage" {
   context = module.this.context
 }
 
-resource "aws_s3_bucket" "default" {
-  count = local.bucket_enabled ? 1 : 0
-
-  #bridgecrew:skip=BC_AWS_S3_13:Skipping `Enable S3 Bucket Logging` check until Bridgecrew will support dynamic blocks (https://github.com/bridgecrewio/checkov/issues/776).
-  #bridgecrew:skip=CKV_AWS_52:Skipping `Ensure S3 bucket has MFA delete enabled` check due to issues operating with `mfa_delete` in terraform
-  bucket        = substr(local.bucket_name, 0, 63)
-  acl           = var.acl
-  force_destroy = var.force_destroy
-  policy        = local.policy
-
-  versioning {
-    enabled    = true
-    mfa_delete = var.mfa_delete
-  }
-
-  server_side_encryption_configuration {
-    rule {
-      apply_server_side_encryption_by_default {
-        sse_algorithm = "AES256"
-      }
-    }
-  }
+module "tfstate_s3_bucket" {
```

Can we call this simply

```suggestion
module "bucket" {
```

or

```suggestion
module "tfstate" {
```